File: code/isPalindrome.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x >= 0 & x<=9)
            return true;
        String str = x + "";
        //System.out.println(str);
        int length = str.length();
        for(int i=0;i<=length/2;i++){
            if(str.charAt(i) != str.charAt(length-1-i))
                return false;
        }
        return true;
        
    }
}

File: code/python/isPalindrome.py
class Solution:
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """

        temp = 0
        z = x
        while x > 0:
            y = x%10;
            temp = temp*10 + y;
            x = x//10;

        if temp == z:
            return True
        else:
            return False
